WordPress/Classes/ViewRelated/Me/App Settings/ExperimentalFeaturesDataProvider.swift

```diff
@@ -5,6 +5,7 @@ class ExperimentalFeaturesDataProvider: ExperimentalFeaturesViewModel.DataProvid
 
     let flags: [OverridableFlag] = [
         FeatureFlag.authenticateUsingApplicationPassword,
+//        FeatureFlag.newGutenberg,
```

Comment:
Commented out code.

Reply:
I put this one in on purpose for when we need it 😅